### ISSUE TITLE
Templating preview.7.20370.3 depends on CSharp >= 3.7.0-3.20269.11 (wrong dependencies)

### ISSUE THREAD

**Issue by timbze** [dotnet/Scaffolding#1390]
## Steps to reproduce:
Using .NET 5.0 preview 7, scaffold any identity pages.
Lint now shows 2 NU1603 warnings:
```
Severity	Code	Description
Warning	NU1603	Microsoft.VisualStudio.Web.CodeGeneration.Templating 5.0.0-preview.7.20370.3 depends on Microsoft.CodeAnalysis.CSharp (>= 3.7.0-3.20269.11) but Microsoft.CodeAnalysis.CSharp 3.7.0-3.20269.11 was not found. An approximate best match of Microsoft.CodeAnalysis.CSharp 3.7.0-3.final was resolved.

Severity	Code	Description
Warning	NU1603	Microsoft.VisualStudio.Web.CodeGeneration.Utils 5.0.0-preview.7.20370.3 depends on Microsoft.CodeAnalysis.CSharp.Workspaces (>= 3.7.0-3.20269.11) but Microsoft.CodeAnalysis.CSharp.Workspaces 3.7.0-3.20269.11 was not found. An approximate best match of Microsoft.CodeAnalysis.CSharp.Workspaces 3.7.0-3.final was resolved.
```

## Expected behavior:
Dependencies should be correct

## Actual behavior:
Seems that wrong dependencies are linked

## dotnet --info
```
.NET SDK (reflecting any global.json):
 Version:   5.0.100-preview.7.20366.6
 Commit:    0684df3a5b

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19041
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.100-preview.7.20366.6\

Host (useful for support):
  Version: 5.0.0-preview.7.20364.11
  Commit:  53976d38b1

.NET SDKs installed:
  3.1.302 [C:\Program Files\dotnet\sdk]
  3.1.400 [C:\Program Files\dotnet\sdk]
  5.0.100-preview.7.20366.6 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.0-preview.7.20365.19 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.0-preview.7.20364.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.6 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.0-preview.7.20366.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Package version of Microsoft.VisualStudio.Web.CodeGeneration.Design - this may be added to your project by scaffolding:
5.0.0-preview.7.20370.3

**Comment by deepchoudhery** [comment]
This will be fixed for 5.0-Preview 8 packages and beyond. Thanks.

**Comment by deepchoudhery** [comment]
fixed for dotnet-aspnet-codegenerator (.NET 8 and 9) + dotnet-scaffold, closing., please use the new 'dotnet-scaffold' tool. Blog [here](https://devblogs.microsoft.com/dotnet/introducing-dotnet-scaffold/)
